### DISCUSSION TITLE
Using NSGA-2 to maximize AUC and sensitivity with respect to VotingClassifier weights

### DISCUSSION THREAD

**Discussion by panos1998** [anyoptimization/pymoo#262] [Q&A]
Hello guys. While  I was searching for nsga-2 python implementation, I found pymoo. So, Iam completely pretty new  to pymoo and relatively new to optimization. Iam wondering how to use nsga-2 to find the best weights for a voting classifier consisting of a  Random forest and  a Logistic Regression. So we have 2 variables [w1, w2] to find their values, which maximize both  the objectives Area Under Curve and Sensitivity.**_Also some constraints are 0<= w1 <=1 and 0<=w2<=1 and w1+w2 =1_**. The voting classifier is  like that 
**_class sklearn.ensemble.VotingClassifier(estimators, *, voting='hard', weights=None, n_jobs=None, flatten_transform=True, verbose=False)_**
Source : https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.VotingClassifier.html
Any ideas how to implement the evaluate() of problem class? Thank you in advance

**Comment by ubeydtalha** [comment]
It's not long since I started using this library, but it's really easy to use.
Idk , did you look documantation , my think your answers are here ^^.
https://pymoo.org/getting_started/part_2.html

Also , you can parallelizate for each population,
https://pymoo.org/problems/parallelization.html#id1

**Reply by panos1998** [reply]
thank you for the additional resources, although I read some documentation,  I did not find this particular example, that seems very expressive, detailed and matches at some point my problem. I will check and response
